dora-parser/src/lexer/position.rs

```diff
@@ -14,6 +14,10 @@ impl Position {
 
         Position { line: l, column: c }
     }
+
+    pub fn invalid() -> Position {
```

Comment:
Should we have this method at all? We are a bit circumventing around the type system and assertions? If we might not have a position we could use `Option<Position>` or if we prefer `Position` we could simply continue to use `Position::new(1, 1)`.

Reply:
I don't care much either way, I was picking a way that would make it immediately obvious that the values are invalid, unlike `Position::new(1, 1)`, which is a valid row/column set that can appear in a real file.